### ISSUE TITLE
Mobile Meta Tag

### ISSUE THREAD

**Issue by Mikeysax** [runyourgym/splurty#1]
On mobile things don't look as good as they could be.

One thing you could look into is to add the "mobile meta tag" to your app, so the responsive effect of Bootstrap actually shows up on Mobile.

Let me know in the comments once you figured that step out :)

**Comment by mattverlaque** [comment]
Resolved. I had a few Bootstrap 3 classes in there, as you mentioned. `img-responsive` was switched out in lieu of `img-fluid`, and `col-xs-offset-1` was switched out with `xs-offset-1`, to specifically address the two examples via email. I ran through the rest of the .html.erb files looking for additional inconsistencies, and I think that I got them all.

The mobile issue has been tested and appears to be functioning as designed - my oversight for not doing proper testing on mobile prior to pushing live.

The mobile meta tag `<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">` has also been added per Bootstrap 4 recommendations, after getting the tip from you here :)

Let me know if that resolves on your end and/or if you see any other issues!

Matt
